storage/namespace.go

```diff
@@ -670,6 +675,17 @@ func (n *dbNamespace) CleanupFileset(earliestToRetain time.Time) error {
 	return multiErr.FinalError()
 }
 
+func (n *dbNamespace) DeleteInactiveFilesets() error {
+	var shardIds []uint32
+	filePathPrefix := n.opts.CommitLogOptions().FilesystemOptions().FilePathPrefix()
+	activeShards := n.getOwnedShards()
+	for _, shard := range activeShards {
+		shardIds = append(shardIds, shard.ID())
+	}
+
+	return n.deleteInactiveFilesetsFn(filePathPrefix, n.id, shardIds)
```

Comment:
We still going to move this into cleanup mgr directly as per other comment thread?

Reply:
yes, that's the plan

Reply:
i'll add a comment in the code